install_config/install/prerequisites.adoc

```diff
@@ -197,14 +197,68 @@ The following must be set up in your environment before OpenShift can be
 installed.
 
 [[prereq-dns]]
-
 === DNS
 
+A fully functional DNS environment is an absolute requirement for
+{product-title} to work correctly. It is not enough to simply add entries for
+hosts in the *_/etc/hosts_* file, because the DNS resolver libraries do not
+query that file. Also, key components of OSE v3 run themselves inside of
+containers. By default, these containers receive their *_/etc/resolv.conf_* DNS
+configuration file from their host. {product-title} then inserts one DNS value
+into the pods (above the node's nameservers values). That value is defined in
+the *_/etc/origin/node/node-config.yaml_* by the `*dnsIP*` parameter. By default
+this will be the IP address of the host node since *dnsmasq* is configured.
+
+As of {product-title} 3.2, *dnsmasq* is automatically configured on all nodes.
+*NetworkManager* is required on the nodes in order to populate
+*dnsmasq* with the DNS IP addresses. The pods use the nodes as their DNS, and
+the nodes forward the requests. By default, *dnsmasq* is configured on the nodes
+to listen on port 53, therefore the nodes cannot run any other type of DNS
+application.
+
+To configure the {product-title} DNS environment:
+
+- (Optionally) xref:../../install_config/install/prerequisites.adoc#wildcard-dns-prereq[configure a wildcard for the router]
+- xref:../../install_config/install/prerequisites.adoc#dns-config-prereq[Complete DNS configuration]
+
+If you do not have a properly functioning DNS environment, you could experience failure with:
+
+- Product installation via the reference Ansible-based scripts
+- Deployment of the infrastructure containers (registry, routers)
+- Access to the {product-title} web console, because it is not accessible via IP address alone
+
+[CAUTION]
+====
+A DNS server cannot be installed on the master nodes because an {product-title}
+master runs its own internal DNS server for service discovery. This internal DNS
+server cannot serve general DNS queries, including the master and nodes'
+hostnames.
+====
```

Comment:
With master nodes using *dnsmasq*  we change the *SKYDNS* that in 3.1 was configure for on port 53 to now listen on 8053 so that  dnsmasq can run on the master since it is also a node.   

```
$ cat /etc/origin/master/master-config.yaml 

dnsConfig:
  bindAddress: 0.0.0.0:8053
  bindNetwork: tcp4
```